### PR TITLE
chore: archive kpng sub-project for SIG Network

### PR DESCRIPTION
**Pull request by shaneutt**
This is a companion issue to https://github.com/kubernetes/org/issues/5054 which archives the [KPNG](https://github.com/kubernetes-sigs/kpng) sub-project.

### CODE REVIEW

**Comment by shaneutt** [comment]
@Priyankasaggu11929 am I reading this right, that there's some kind of problem here with archiving a slack channel?

File: communication/slack-config/sig-network/config.yaml [hunk 1]
```diff
@@ -2,4 +2,3 @@ channels:
   - name: sig-network-gateway-api
     id: CR0H13KGA
   - name: sig-network-policy-api
```
Comment (Priyankasaggu11929) [inline communication/slack-config/sig-network/config.yaml:4]: > Deleting channels is not permitted, but a channel can be archived by specifying archived: true, or unarchived by removing it (or specifying false).

ref: https://github.com/kubernetes/community/blob/9b7157ce64a0948de6ddd76ca4f51e760a4138b6/communication/slack-config/README.md#channels

```suggestion
  - name: sig-network-policy-api
  - name: sig-network-kpng
    archived: true
```

**Review by Priyankasaggu11929 — CHANGES_REQUESTED** [review]

File: communication/slack-config/sig-network/config.yaml (hunk 1 above)
Comment (shaneutt) [reply communication/slack-config/sig-network/config.yaml:4]: Awesome thank you!

**Comment by Priyankasaggu11929** [comment]
/lgtm
/approve

/hold
for any further review from SIG Network leads

**Comment by thockin** [comment]
hold for others

/lgtm
/approve
/hold

**Pull request merged**